bin-src/main.ts

```diff
@@ -77,12 +77,12 @@ export async function runAll(ctx) {
   }
 }
 
-export async function runBuild(ctx: Context) {
+export async function runBuild(ctx: Context, extraOptions?: Partial<Options>) {
   ctx.log.info('');
   ctx.log.info(intro(ctx));
 
   try {
-    ctx.options = await getOptions(ctx);
+    ctx.options = await getOptions(ctx, extraOptions);
```

Comment:
```suggestion
    ctx.options = await getOptions(ctx);
    if (extraOptions) ctx.options = { ...ctx.options, ...extraOptions };
```
This makes it more apparent that "all bets are off" at this point.